tools/bin/make-filesystem-image

```diff
@@ -1,10 +1,10 @@
-#!/bin/bash
+#!/usr/bin/env bash
 
-set -e
+set -euo pipefail
 
 IMG="$1"
 
 mkdir -p "$(dirname "$IMG")"
 dd if=/dev/zero of="$IMG" bs=100M count=24
-yes | mkfs -t ext3 -m 1 "$IMG"
```

Comment:
The `yes` command outputs `y` strings repeatedly until it is killed by the `SIGPIPE` when mkfs exits. Its non-clean exit causes the overall script to error due to `pipefail` mode. 

We could disable `pipefail` for this command only, however the use of `yes` was a [workaround](https://askubuntu.com/questions/428474/automatic-say-yes-to-make-ext3-file-system-on-disk-with-mkfs-ext3) for `mkfs` needing a confirmation prompt in certain cases - and trying mkfs now without the `yes` shows it no longer prompts, so it's redundant.

(I tracked the first usage of `yes` with `mkfs` to: https://github.com/heroku/capturestack/commit/ac525cac9291aa376f16e6f5ac460c9a77cf4d2f - that repo was presumably the inspiration for https://github.com/heroku/stack-image-tools/pull/3 which was then eventually vendored here.)